src/components/secret/AddSecret.jsx

```diff
@@ -185,12 +185,12 @@ const AddSecret = props => {
           {secretType === "file" && (
             <Form.Item>
               {getFieldDecorator(`file[${k}].value`, {
-                rules: [{ required: true, message: "Please upload a file" }]
+                rules: [{ required: true, message: "Please input file contents" }]
```

Comment:
Great! I liked that you actually checked even the messages that you weren't supposed to and fixed them!